listenbrainz/webserver/static/css/brainzplayer.less

```diff
@@ -239,9 +247,21 @@
 			border-right: 2px solid @dark-color;
 		}
 		&:hover{
+			transition: height, 0.3s;
```

Comment:
Yes to the transition, but:
1. it needs to be defined in the parent .progress, not on hover (otherwise it's only applied when you're hovering)
2. I'd shorten the animation time just a smidge: 0.2s

Reply:
AH! And I was wondering why wasn't the progressbar animating after hovering out the mouse 😅

Reply:
Yeah, I've done that one many times before…

Reply:
Haha 😆 
This one's fixed in the upcoming changes :)